src/app/containers/Navigation/index.amp.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import Navigation from './index.amp';
+
+const dropdownTestId = 'dropdown';
+const scrollableTestId = 'scrollable-list';
+
+const scrollableListItems = (
+  <ul data-testid={scrollableTestId}>
+    <li>List Items</li>
+  </ul>
+);
+
+const dropdownListItems = <div data-testid={dropdownTestId}>Dropdown</div>;
+
+const navigation = (
+  <Navigation
+    scrollableListItems={scrollableListItems}
+    dropdownListItems={dropdownListItems}
+    skipLinkText="skip link"
```

Comment:
We don't need to pass the `Skip link` to the Navigation anymore, since it will be handled by the `Brand`.

Reply:
Wondering if removing skipLink from the Navigation should be done on [this](https://github.com/bbc/simorgh/pull/4948) PR?

Reply:
You can remove it from the `NavigationContainer`, but still, we don't need to pass it in this PR.

Reply:
Good spot - I missed that this was still being passed.